### PR TITLE
Console release notes 2022-09-28

### PR DESCRIPTION
**Pull request by ololobus**

### CODE REVIEW

File: content/release-notes/2022-09-28.md
Comment (danieltprice) [inline content/release-notes/2022-09-28.md:8]:
```suggestion
- Console: Added the ability to select access scopes and projects on the OAuth consent page.
```

File: content/release-notes/2022-09-28.md
Comment (danieltprice) [inline content/release-notes/2022-09-28.md:9]:
```suggestion
- Console: Added a `protected: boolean` flag to the roles public API response.
```

File: content/release-notes/2022-09-28.md
Comment (danieltprice) [inline content/release-notes/2022-09-28.md:11]:
```suggestion
- Console: Changed the operations id from a serial number to a UUID type in the public API.
```

File: content/release-notes/2022-09-28.md
Comment (danieltprice) [inline content/release-notes/2022-09-28.md:12]:
```suggestion
- Control Plane: Added an internal mechanism for relocating projects between pageservers.
```

File: content/release-notes/2022-09-28.md
Comment (danieltprice) [inline content/release-notes/2022-09-28.md:15]:
```suggestion
- Control Plane: Split he monolithic project object into project, branches, and endpoints in the internal database schema.
```

File: content/release-notes/2022-09-28.md
Comment (danieltprice) [inline content/release-notes/2022-09-28.md:19]:
```suggestion
- UI: Fixed a failure that occurred after receiving an empty response in the SQL Editor.
```

File: content/release-notes/2022-09-28.md
Comment (danieltprice) [inline content/release-notes/2022-09-28.md:20]:
```suggestion
- UI: Fixed an issue to ensure that table names are reset when switching between schemas in the Tables UI.
```

File: content/release-notes/2022-09-28.md
Comment (danieltprice) [inline content/release-notes/2022-09-28.md:21]:
```suggestion
- Control Plane: Fixed the passwordless proxy connection for the most recently created project.
```

**Review by danieltprice — APPROVED** [review]
These are great. Just very minor edits.

**Comment by ololobus** [comment]
Committed all suggestions, so I think it's ready to go and is waiting for an approval @stepashka @danieltprice

File: content/release-notes/2022-09-28.md
Comment (danieltprice) [reply content/release-notes/2022-09-28.md]: Ho @ololobus How about:
"Fixed an issue that occurred when creating a new project using a passwordless proxy connection.

**Review by stepashka — DISMISSED** [review]

File: content/release-notes/2022-09-28.md
Comment (ololobus) [reply content/release-notes/2022-09-28.md]: Sounds good to me! Reworded a bit, though

**Pull request merged**
